x/session/keeper/query_get_session_test.go

```diff
@@ -75,6 +76,7 @@ func TestSession_GetSession_Success(t *testing.T) {
 
 func TestSession_GetSession_Failure(t *testing.T) {
 	keeper, ctx := keepertest.SessionKeeper(t)
+	ctx = ctx.WithBlockHeight(100) // provide a sufficiently large block height to avoid errors
```

Comment:
What does this actually do?

Reply:
It literally just sets a variable to a value for when we test things.

```go
// WithBlockHeight returns a Context with an updated block height.
func (c Context) WithBlockHeight(height int64) Context {
	newHeader := c.BlockHeader()
	newHeader.Height = height
	return c.WithBlockHeader(newHeader)
}
```